Core.DomainModel/GDPR/DataProcessingRegistration.cs

```diff
@@ -195,5 +196,9 @@ private Maybe<DataProcessingRegistrationRight> GetRight(DataProcessingRegistrati
         public int? ReferenceId { get; set; }
 
         public virtual ExternalReference Reference { get; set; }
+
+        public YearMonthIntervalOption? OversightInterval { get; set; }
```

Comment:
Her skal den være null modsat på klientkontrakten, så her skal du bare lade den være nullable. Men vi ønsker ikke at man fra klienten kan sætte null. Ønsker man som bruger at sætte "blank" vil vi bruge "undecided". Derved kan vi ved senere analyse se om brugeren rent faktisk har forholdt sig til værdien eller ej modsat hvis null var ok og vi ikke havde mulighed for at sende "undecided" ind